scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -81,6 +82,43 @@ class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig)
     case _ => sys.error(s"unsupported tree $tree")
   }
 
+  def symbolOf(signature: Signature): Option[String] =
```

Comment:
This method needs to either: 1) have a better name, or 2) get inlined into isSameOrAlias.